apptools/apptools-windows-tests/README

```diff
@@ -0,0 +1,35 @@
+## Introduction
+
+This test suite is for apptools-windows-tests
+1. ./allpairs.py is create pkgName.py and report directory for package name
+
+## Precondition
+
+1. The node.js, Android SDK, JDK, apache ant, and git must be functional
```

Comment:
Android SDK, JDK, apache ant are prerequisite for android backend, not for windows